src/client/components/ResultMessageModalWindow/Result.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function Result({ correctAnswer }) {
```

Comment:
Try to think carefully about the names you give to functions and variables/constants. What does Result mean? What result?

Reply:
name of component has changed